### DISCUSSION TITLE
piccolo user subcommands show even if the user app is not registered

### DISCUSSION THREAD

**Discussion by scott2b** [piccolo-orm/piccolo#867] [General]
I am wondering if this is expected behavior. The following items show even though I have not registered `piccolo.apps.user.piccolo_app` in my AppRegistry:

```
$ piccolo
...

user create
Create a new user.

user change_password
Change a user's password.

user change_permissions
Change a user's permissions.
```

Ideally, it seems like these commands would only be available if I am using the user app. Perhaps there is a way to explicitly disable them?

**Comment by scott2b** [comment]
Ok, I believe the [hard-coded inclusion of the user_config](https://github.com/piccolo-orm/piccolo/blob/master/piccolo/main.py#L73) in the main cli to be the culprit here.

I have, for the time being, simply opted to create my own copy of the main.py cli with the user config removed. I have only tested it so far with the migrations subcommand, but that works, so going for the moment with the assumption that things will work in general.

One weird caveat: It seems that I cannot have my custom User in a package called user. I can name the app user, and that is ok, but if it is in my code under `myapp.user`, the commands will not show up. I have placed it instead under `myapp.users` and it seems to be working fine.

**Reply by dantownsend** [reply]
You're right - the builtin `user` app is auto registered. Your solution sounds like a good one for now. Ideally there would be an option in `piccolo_conf.py` which lets you disable some of the auto registered apps.
